Mammoth/Include/TSEUniverse.h

```diff
@@ -372,6 +372,7 @@ class CUniverse
 		void FireOnGlobalUniverseSave (void) { m_Design.FireOnGlobalUniverseSave(); }
 		void FlushStarSystem (CTopologyNode *pTopology);
 		void GenerateGameStats (CGameStats &Stats);
+		CAccessibilitySettings* GetAccessibilitySettings(void) { return &m_AccessabilitySettings; }
```

Comment:
I think returning a reference is better here; then the caller doesn't have to worry about NULL.

Reply:
refactored this to use a reference instead